docs/UserGuide.md

```diff
@@ -122,19 +127,19 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
-### Deleting a person : `delete`
+### Deleting an existing task : `delete`
 
-Deletes the specified person from the tr4cker.
+You can delete an existing task from the task list by providing the 
+index number of the task that you want to delete.
 
 Format: `delete INDEX`
 
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
+* Deletes the task at the specified `INDEX`.
+* The index refers to a valid index number shown in the displayed task list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd task in tr4cker.
-* `find Betsy` followed by `delete 1` deletes the 1st task in the results of the `find` command.
```

Comment:
this function works in Tr4cker too. maybe we should keep the corresponding version of this line?

Reply:
Yes you're right.